pootle/apps/pootle_app/management/commands/config.py

```diff
@@ -0,0 +1,236 @@
+                "VALUE as json"))
+        parser.add_argument(
+            '-a', '--append',
+            action='store',
+            dest="append",
+            metavar=("KEY", "VALUE"),
+            nargs=2,
+            help=_("Append config with this value"))
+        parser.add_argument(
+            '-c', '--clear',
+            action='append',
+            dest='clear',
+            metavar="KEY",
+            help=_("Clear config with this key"))
+
+    def print_config_item(self, item, object_field=None):
+        if item.content_type:
+            ct = ".".join(item.content_type.natural_key())
+            if item.object_pk:
+                if object_field:
+                    pk = getattr(
+                        item.content_object,
+                        object_field)
+                else:
+                    pk = item.object_pk
+                item_name = "%s[%s]" % (ct, pk)
+            else:
+                item_name = ct
+        else:
+            item_name = "Pootle"
+        self.stdout.write(
+            "{: <25} {: <25} {: <30}".format(
+                item_name, item.key, self.repr_value(item.value)))
+
+    def get_conf(self, **kwargs):
+        if kwargs["object"] and not kwargs["content_type"]:
+            raise CommandError(
+                _("You must set --content-type (-t) when using "
+                  "--object-pk (-o)"))
+
+        if kwargs["content_type"]:
+            if "." not in kwargs["content_type"]:
+                raise CommandError(
+                    _("content_type should be set with $app_label.$model_name"))
+            parts = kwargs["content_type"].split(".")
+            model_name = parts[-1]
+            app_label = ".".join(parts[:-1])
+            try:
+                ct = ContentType.objects.get(
+                    app_label=app_label,
+                    model=model_name)
+            except ContentType.DoesNotExist as e:
+                raise CommandError(e)
+            ct_model = ct.model_class()
+            if kwargs["object"]:
+                if kwargs["object_field"]:
+                    model_query = {kwargs["object_field"]: kwargs["object"]}
+                else:
+                    model_query = dict(pk=kwargs["object"])
+                catch_exc = (
+                    ct_model.DoesNotExist,
+                    ct_model.MultipleObjectsReturned,
+                    FieldError)
+                try:
+                    instance = ct_model.objects.get(**model_query)
+                except catch_exc as e:
+                    raise CommandError(e)
+                return config.get(ct_model, instance=instance)
+            return config.get(ct_model)
+        return config.get()
+
+    def print_no_config(self, **kwargs):
+        self.stdout.write(_("No configuration found"))
+
+    def check_incompatible_flags(self, **kwargs):
+        mutually_incompatible = ["get", "set", "append", "list", "clear"]
```

Comment:
mmm, just saw this after commenting above about using argparse mutual exclusive group.